04 Research Environment/03 Datasets/11 Crypto Futures/04 Get Historical Data.html

```diff
@@ -0,0 +1,163 @@
+<p>You need a <a href="/docs/v2/research-environment/datasets/crypto-futures#03-Create-Subscriptions">subscription</a> before you can request historical data for a security. On the time dimension, you can request an amount of historical data based on a trailing number of bars, a trailing period of time, or a defined period of time. On the security dimension, you can request historical data for a single contract, a subset of the contracts you created subscriptions for in your notebook, or all of the contracts in your notebook.</p>
```

Comment:
Please avoid these sentence fragments I think they're just more confusing / not needed:
"On the time dimension,"
"On the security dimension,"